interpreter/cling/lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -286,6 +287,9 @@ Error RTDynamicLibrarySearchGenerator::tryToGenerate(
 
 namespace cling {
 
+///\brief Creates JIT event listener to allow profiling of JITted code with perf
```

Comment:
Could call out in a comment where this is implemented? E.g. just
```suggestion
///\brief Creates JIT event listener to allow profiling of JITted code with perf.
/// Implemented in `PerfJITEventListener.cpp`
```